doc/tools/docgram/common.edit_mlg

```diff
@@ -1088,7 +1088,6 @@ simple_tactic: [
 | EDIT "simple" "injection" ADD_OPT destruction_arg
 | REPLACE "instantiate" "(" ident ":=" lglob ")"
 | WITH "instantiate" OPT ( "(" ident ":=" lglob ")" )
```

Comment:
These 2 lines should be removed, too.  The effect of these plus the DELETE was to merge 2 productions.  No merge is needed if you delete the no-arg instantiate.